packages/scroll-sync/README.md

```diff
@@ -0,0 +1,62 @@
+# @infinum/scroll-sync
+
+A component that will sync the scroll between multiple scroll containers
+
+## Installation
+
+```sh
+yarn add @infinum/scroll-sync
+# or
+npm i @infinum/scroll-sync
```

Comment:
We should name it  `@infinum/react-scroll-sync`

Reply:
Because if we want to release vanilla scorl-sync in the future we would not be able release it under `@infinum` org prefix. That lib doesn't have to be a part of react-nuts-and-bolts monorepo.